scripts/make.py

```diff
@@ -577,10 +577,10 @@ def _check_output(output_type: str, smoketest_dir: os.DirEntry,
                 expected_output_path = os.path.join(smoketest_path,
                                                     output_type)
                 try:
-                    expected_output = open(expected_output_path)
-                    expected_output_str = [
-                        s.rstrip('\n') for s in expected_output.readlines()
-                    ]
+                    with open(expected_output_path) as expected_output:
```

Comment:
`pylint` had the following:

`scripts/make.py:580:20: R1732: Consider using 'with' for resource-allocating operations (consider-using-with)`